test/app/lib/world/GoalManager.spec.coffee

```diff
@@ -8,6 +9,35 @@ describe('GoalManager', ->
   stayMapGoal =  {name: 'Stay here', keepFromLeavingOffSide: {who: 'Yall'}, id: 'id'}
   getItemGoal = {name: 'Mine', getItem: {who: 'Grabby', itemID: 'Sandwich'}, id: 'id'}
   keepItemGoal = {name: 'Not Yours', keepFromGettingItem: {who: 'Grabby', itemID: 'Sandwich'}, id: 'id'}
+  # Very simple session mocking
+  session = {
```

Comment:
Nitpick: You can also use utility method to create level session - `factories.makeLevelSession` (https://github.com/codecombat/codecombat/blob/master/test/app/factories.coffee)